pkg/registry/manifest/manifest.go

```diff
@@ -53,7 +53,7 @@ func extractImageAndTag(imageName string) (string, string) {
 		parts := strings.Split(imageName, ":")
 		if len(parts) > 2 {
 			img = fmt.Sprintf("%s%s", parts[0], parts[1])
-			tag = parts[3]
+			tag = parts[2]
 		} else {
 			img = parts[0]
 			tag = parts[1]
```

Comment:
```suggestion
			return parts[0], parts[1]
```

should be sufficient. please check with the test suite before merging.

Reply:
Same type of change should be applied on 62:65 as well, as in switching to a return.